src/HighlightGenerator.js

```diff
@@ -8,9 +9,20 @@ HighlightGenerator.highlightMatches = function highlightMatches(message, warning
     var rectsToHighlight = rangeToHighlight.getClientRects();
     for (var i = 0; i < rectsToHighlight.length; i++) {
       var highlightNode = HighlightGenerator.highlightMatch(rectsToHighlight[i], parentRect);
-      highlightNode.title = message;
+      var messageNode = HighlightGenerator.generateHighlightNode();
+      var scroll = HighlightGenerator.generateScroll();
+      var coords = HighlightGenerator.transformCoordinatesRelativeToParent(rectsToHighlight[i], parentRect);
       highlightNode.className = warningClass;
-      parentNode.appendChild(highlightNode);
+      highlightNode.setAttribute('triggerHeight', rectsToHighlight[i].height);
+      highlightNode.style.zIndex = parentRect.height - coords.top + 200;
+      messageNode.className = "jns-message";
+      messageNode.innerHTML = message;
+      var windowWidth = Math.max(document.documentElement.clientWidth, window.innerWidth || 0);
+      var messageRight = (rectsToHighlight[i].left + scroll.left) + MESSAGE_OFFSET;
+      if(messageRight > windowWidth) {
+          messageNode.style.left = (windowWidth - messageRight) + 'px';
+      }
+      parentNode.appendChild(highlightNode).appendChild(messageNode);
```

Comment:
If I have two compose windows, I see weird scrollbars on one of the windows but not the other:
![image](https://user-images.githubusercontent.com/1520763/79371003-42368280-7f19-11ea-8f31-1068b13c3bf3.png)